data/data/bootstrap/vsphere/files/etc/NetworkManager/dispatcher.d/30-local-dns-prepender.template

```diff
@@ -17,6 +17,11 @@ case "$STATUS" in
             done
         fi
 EOF
+    
+    if [[ ! -e /etc/resolv.conf ]] || ! grep -q nameserver /etc/resolv.conf; then
+        cp /var/run/NetworkManager/resolv.conf /etc/resolv.conf
```

Comment:
I reviewed that the default `cp` command would create or overwrite `/etc/resolv.conf` if it does not exist or does not contain "nameserver".

Nit: can we have this template pass shellcheck?